tests/mock-ui/mock-ui.cpp

```diff
@@ -25,8 +25,7 @@
 
 WebEidUI* WebEidUI::createAndShowDialog(const CommandType)
 {
-    static MockUI instance;
-    return &instance;
+    return new MockUI;
```

Comment:
This leaks, no parent nor `WA_DeleteOnClose`. Should you add `WA_DeleteOnClose`?